### DISCUSSION TITLE
コミットメッセージの文章を日本語と英語のどちらに統一するか

### DISCUSSION THREAD

**Discussion by siso001** [lgtm-factory/lgtm-factory#22] [未分類]
#13 
[資料](https://zenn.dev/itosho/articles/git-commit-message-2023)ではこのように書かれていました。

> **日本語か？英語か？**
> 統一されていることが大切なので、チームでコンセンサスが取れていれば、どちらでも（もちろん、日本語や英語以外でも）良いと思いますが、僕の場合、ドメスティックなチーム、ドメスティックなサービスであれば日本語で書くことが多いです。  
> 
> 過去、張り切って英語で書いていたこともあったのですが、ネイティブじゃないのでどうしても時間がかかったり、急いでいると雑に「Fix a sorting bug」 的な文法も怪しいカーディナリティの高いコミットメッセージを書いたりしてしまうことがあり、本末転倒だなと思って英語で書くことは少なくなりました。  
> 
> もちろん、グローバルなチームでは英語で書いたほうがいいケースがほとんどだと思いますし、英語が書けて損はないので、英語を採用すること自体は全く問題ないですが、ネイティブじゃない場合は、自分だけではなく、チーム全員でそれなりの覚悟が必要かなと思います。

日本語が無難に分かりやすいんじゃないの？と思っていますが、自分は共同開発の経験がないので皆さんの意見を聞きたいです！

**Comment by kazzyfrog** [comment]
自分は、日本語がいいかなと思います！

理由としては、チーム開発初心者？のメンバーを中心に、
色々学びながら取り組むプロジェクトだと思っているので、分かりやすさ重視がいいかなと！👍

**Reply by kazzyfrog** [reply]
メモ：
グローバルな企業であったり、
世界中の開発者の参加を想定したオープンソースプロジェクトなどは、英語を使うことが多いです！

一方、日本の会社とかで、日本語でやり取りが行われるチームであれば、日本語で書くことも割と一般的ですよ！

**Reply by siso001** [reply]
ご意見ありがとうございます！
なるほど、日本語で書くことも一般的なんですね。勉強になります🤘

**Comment by kagomen** [comment]
自分も日本語が良いかなと思います！
英語の勉強？などがプロジェクトの目的に含まれてたら、コミットメッセージを英語に指定することで英語の勉強にもなりそうですね！

**Reply by siso001** [reply]
確かに！目的によっては英語もアリですね
ありがとうございます😊
